notifications.bs

```diff
@@ -83,9 +83,9 @@ initially false. When true, indicates that the end user should be alerted after
 an existing notification.
 
 <p>A <a>notification</a> has an associated
-<dfn for=notification id=silent-preference-flag>silent preference</dfn> boolean, which is initially
-null. When true, indicates that no sounds or vibrations should be made. When null, indicates that
-producing sounds or vibrations should be left to platform conventions.
+<dfn for=notification id=silent-preference-flag>silent preference</dfn> nullable boolean, which is
```

Comment:
While interesting, I'd prefer we stick with "null or boolean" or "boolean or null" for the type. We're not quite happy with IDL's ?-convention so I don't think we should start using it in prose.

Reply:
Makes sense.